src/climate_learn/training/train.py

```diff
@@ -71,5 +71,9 @@ def __init__(
     def fit(self, model_module, data_module):
         self.trainer.fit(model_module, data_module)
 
-    def test(self, model_module, data_module):
-        self.trainer.test(model_module, data_module)
+    def test(self, model_module, data_module, checkpoint_path=""):
```

Comment:
IMO if `checkpoint_path` is an optional argument, it should default to `None`, not the empty string.